packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import flexStyles from '@patternfly/patternfly-next/utilities/Flex/flex.css';
+import displayStyles from '@patternfly/patternfly-next/utilities/Display/display.css';
+import alignmentStyles from '@patternfly/patternfly-next/utilities/Alignment/alignment.css';
+import spacingStyles from '@patternfly/patternfly-next/utilities/Spacing/spacing.css';
+import { Form, FormGroup, ActionGroup } from '../Form';
+import { TextInput } from '../TextInput';
+import { Button } from '../Button';
+import { Checkbox } from '../Checkbox';
+
+const propTypes = {
+  /** Additional classes added to the LoginBox Body */
+  className: PropTypes.string,
+  /** Label for the Username Input Field */
+  usernameLabel: PropTypes.string,
+  /** Value for the Username */
+  usernameValue: PropTypes.string,
+  /** Function that handles the onChange event for the Username */
+  onChangeUsername: PropTypes.func,
+  /** Helper Text for the Username Input Field */
+  usernameHelperText: PropTypes.string,
+  /** Helper Text for the Username Input Field when it is invalid */
+  usernameHelperTextInvalid: PropTypes.string,
+  /** Flag indicating if the Username is valid */
+  isValidUsername: PropTypes.bool,
+  /** Label for the Password Input Field */
+  passwordLabel: PropTypes.string,
+  /** Value for the Password */
+  passwordValue: PropTypes.string,
+  /** Function that handles the onChange event for the Password */
+  onChangePassword: PropTypes.func,
+  /** Helper Text for the Username Input Field */
+  passwordHelperText: PropTypes.string,
+  /** Helper Text for the Password Input Field when it is invalid */
+  passwordHelperTextInvalid: PropTypes.string,
+  /** Flag indicating if the Password is valid */
+  isValidPassword: PropTypes.bool,
+  /** Label for the Log in Button Input */
+  loginButtonLabel: PropTypes.string,
+  /** Flag indicating if the Login Button is disabled */
+  isLoginButtonDisabled: PropTypes.bool,
+  /** Function that is called when the Login button is clicked */
+  onLoginButtonClick: PropTypes.func,
+  /** Label for the Remember Me Checkbox that indicates the user should be kept logged in */
+  rememberMeLabel: PropTypes.string,
+  /** Flag indicating if the remember me Checkbox is checked. */
+  isRememberMeChecked: PropTypes.bool,
+  /** Function that handles the onChange event for the Remember Me Checkbox */
+  onChangeRememberMe: PropTypes.func,
+  /** Aria Label for the Remember me checkbox */
+  rememberMeAriaLabel: PropTypes.string.isRequired
```

Comment:
should only be required if the rememberMeLabel is also passed in

Reply:
The Checkbox component requires the Aria Label

Reply:
Yes, but in this form having a checkbox with no label would not make sense, so I think if you require the aria-label you should also require the rememberMeLabel. But i could see consumers maybe not wanting to have this checkbox, so perhaps add the logic here, if rememberMeAriaLabel is passed in, then require the aria-label as well. And render the Checkbox conditionally if rememberMeLabel is used.